model.py

```diff
@@ -59,11 +59,11 @@ class Responder(db.Model):
   # The Device Model ID field from DEVICE_INFO
   device_model_id = db.IntegerProperty()
   # The DEVICE_MODEL_DESCRIPTION
-  model_description = db.StringProperty(required=True)
+  model_description = db.StringProperty(default='')
```

Comment:
I've changed from required, to default='' as per https://github.com/OpenLightingProject/rdm-app/pull/50 , but I'm wondering, why don't we default to None instead, so we can filter/search on responders with no model description?

Reply:
Makes sense.

Reply:
Okay, I've started moving to None and the related kick off in the templates.

Any thoughts on what name a device with no model description should show in http://rdm.openlighting.org/model/browse ? Currently it's just the RDM ID and some words (I wonder if it should include the Manufacturer ID too).

Reply:
That sounds reasonable.